packages/vaadin-grid/test/accessibility.test.js

```diff
@@ -2,8 +2,7 @@ import { expect } from '@esm-bundle/chai';
 import { fixtureSync, nextFrame } from '@vaadin/testing-helpers';
 import '@vaadin/vaadin-template-renderer';
 import { flushGrid } from './helpers.js';
-import '../vaadin-grid.js';
-import '../vaadin-grid-column-group.js';
+import '../all-imports';
```

Comment:
nit: let's use `.js` extension for consistency.

Reply:
Done